packages/manager/apps/pci-instances/src/pages/instances/create/CreateInstance.page.tsx

```diff
@@ -85,6 +100,45 @@ const CreateInstance: FC = () => {
                 {t('creation:pci_instance_creation_name_rule')}
               </Text>
             </section>
+            <section className="flex flex-col w-full mb-8">
+              <Text preset="heading-2">
+                {t('creation:pci_instances_creation_quantity_title')}
+              </Text>
+              <div className="pt-3 pb-4">
+                <FormField>
+                  <FormFieldLabel>
+                    {t('creation:pci_instances_creation_quantity_label')}
+                  </FormFieldLabel>
+                  <Quantity
+                    {...register('quantity', { required: true })}
+                    onValueChange={({ value }) => {
+                      setValue('quantity', Number(value));
```

Comment:
For [Quantity](url), I'm not sure that it's the way to handle it as it's a custom component.
I think we have to deal with RHF `Controller` (https://react-hook-form.com/docs/usecontroller/controller), maybe you could check with our RHF experts ;) (@SimonChaumet and @ppprevost  if I'm correct?)

Reply:
Yes, either you use the register only, or you use the Controller component to wrap it. setValue should only be used when controlling values with side effects

Also I think it is a good idea to create pre-wrapped react-hook-form generic fields in a separate folder (like I did here : https://github.com/ovh/manager/pull/18023/files#diff-51c94ff4882234eee80e8d1f435360f8da2c925cca72abddf5c2df41f7920df1)